ArduPlane/ArduPlane.cpp

```diff
@@ -532,7 +532,7 @@ void Plane::set_flight_stage(AP_Vehicle::FixedWing::FlightStage fs)
 
     if (fs == AP_Vehicle::FixedWing::FLIGHT_ABORT_LAND) {
         gcs().send_text(MAV_SEVERITY_NOTICE, "Landing aborted, climbing to %dm",
-                          auto_state.takeoff_altitude_rel_cm/100);
+                        int(auto_state.takeoff_altitude_rel_cm/100));
```

Comment:
Int?

Reply:
Yes, `int`.  That's *explicitly* what `%d` wants.

If you *do* want to use the `stdint` types in format strings then you need to use the stuff @OXINARF was talking about elsewhere - the PRI stuff.  In this case it would look like this, if I'm right:

```
gcs().send_text(MAV_SEVERITY_NOTICE, "Landing aborted, climbing to %" PRIi32 "m",
                          auto_state.takeoff_altitude_rel_cm/100);
```

We crossed this in the last devcall.  We decided on the approach I've used in this PR.

Reply:
I wasn't there though and I definitely didn't agree to replacing (int) with int()..

Reply:
@rmackay9 tridge has a preference for the functional form, I believe.

I usually go with `(int)` in the case of a single parameter - but in this case we would have changed

`auto_state.takeoff_altitude_rel_cm/100)`

into `(int)(auto_state.takeoff_altitude_rel_cm/100))`

rather than what's in this PR:

int(auto_state.takeoff_altitude_rel_cm/100))

I find the last more readable myself.